.github/workflows/run-acceptance-tests.yml

```diff
@@ -53,6 +53,41 @@ jobs:
     secrets: inherit
     with:
       version: ${{ needs.prerequisites.outputs.version }}
+  combine_build_artifacts:
```

Comment:
Perhaps you know this and are just experimenting but FYI these are auto-generated files, and the team is in the middle of refactoring these workflows elsewhere.

Reply:
yeah I'm just experimenting